src/transactionsTable/transactionsTable.tsx

```diff
@@ -54,22 +56,24 @@ export const TransactionsTable: React.FC<TransactionsTable> = props => {
   const columns = [
     {
       name: "transactions",
-      title: "transactions",
+      title: <>transactions</>,
```

Comment:
What is the reason to wrap a string in React.Fragment?
The type of `title` field can accept strings as well.

Reply:
It’s the SortableColumn interface. It awaits for react.Node cell: (rowIndex: number) => React.ReactNode. I wanted to avoid making changes to SortedTable in this branch, just clean usage.